third_party/move/move-vm/types/src/loaded_data/struct_name_indexing.rs

```diff
@@ -82,6 +84,11 @@ impl StructNameIndexMap {
         let forward_key = struct_name.clone();
         let backward_value = Arc::new(struct_name.clone());
 
+        let mut hasher = blake3::Hasher::new();
```

Comment:
In theory we can compute the hash when we pass StructIdentifier here, and module construction builds it and pass to all structs?

Reply:
What you described is the ideal thing I wanted to do. Sounds like it may not be that hard? Let me try it out.